pandas/tests/indexes/interval/test_interval.py

```diff
@@ -616,6 +624,14 @@ def test_get_indexer_length_one(self, item, closed):
         expected = np.array([0] * len(item), dtype='intp')
         tm.assert_numpy_array_equal(result, expected)
 
+    @pytest.mark.parametrize('index', [
+        pd.interval_range(0, 1),
+        pd.IntervalIndex.from_tuples([(1, 3), (2, 4), (0, 2)])
+    ])
+    def test_get_indexer_errors(self, index):
+        expected = np.array([-1], dtype='intp')
+        assert tm.assert_numpy_array_equal(index.get_indexer(['gg']), expected)
```

Comment:
Can you here also test multiple values and a mixture of values? Like `get_indexer(['a', 'b']` and `get_indexer([1, 'a'])`

Reply:
Can you do this one?

Reply:
Sure will look into this over the weekend.

Reply:
see 02127ff